sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -38,3 +38,43 @@ If no `options` parameter is supplied, the report is exported using the default
 
 Refreshes the data for the report object that is controlled by the
 `ObjectHandle`.
+
+### getSelectedData(options?: {formatData: boolean | "datesOnly"}): ReportObjectResultData[]
+
+Returns a user's selection data from the visual. Returns an empty array if the visual has no selections.
+
+#### Arguments
+
+`options` is an optional options bundle for customizing the returned data. The following options are supported:
+
+- `formatData` for specifying the format of the returned data. It can be passed the following values:
+  - `true` to receive all formatted data
```

Comment:
'true' receives all formatted data

Reply:
I swapped out 'receives' for 'returns' in these 3 bullet points, since it's the user who is 'receiving' the data instead of the variable so 'returns' feels more accurate now